src/main/scala/com/target/data_validator/validator/MinMaxChecks.scala

```diff
@@ -0,0 +1,53 @@
+package com.target.data_validator.validator
+
+import com.target.data_validator.JsonUtils.debugJson
+import com.target.data_validator.ValidatorError
+import io.circe.Json
+
+trait MinMaxChecks { this: ValidatorBase =>
+  val minValue: Option[Json]
+  val maxValue: Option[Json]
+  val inclusive: Option[Json]
+
+  lazy val minMaxList: List[Json] = (minValue :: maxValue :: Nil).flatten
```

Comment:
Is list the correct data structure for these?

Reply:
Probably not but I didn't change it from what it was before. As a list, it makes `checkMinLessThanMax` a little easier to read.